bin/node/runtime/pangolin/src/lib.rs

```diff
@@ -303,19 +390,15 @@ type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
 type Ring = Balances;
 
-type NegativeImbalance = <darwinia_balances::Module<Runtime, RingInstance> as Currency<
-	<Runtime as frame_system::Config>::AccountId,
->>::NegativeImbalance;
-
 /// This runtime version.
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("Pangolin"),
 	impl_name: create_runtime_str!("Pangolin"),
 	authoring_version: 1,
-	spec_version: 13,
+	spec_version: 20,
```

Comment:
Update spec_version from 13 to 20 directly, why not 14?

Reply:
This's a re-genesis. Not runtime upgrade.